lib/iris/__init__.py

```diff
@@ -114,6 +115,7 @@ def callback(cube, field, filename):
     "load_cube",
     "load_cubes",
     "load_raw",
+    "main",
```

Comment:
`main()` was an OK name for the function within `iris.experimental.ugrid.cli`, but given the wider usage could we come up with something less vague? `cli`, or `cli_main`, or something else?